src/pages/trails/ballots/components/BallotListItem.vue

```diff
@@ -135,8 +135,10 @@ div
 
       div.text-section-row
         div.statics-section-item.section-item-btn
-          btn(
-            :labelText="mainButtonTextSmall"
+          q-btn(
+            no-caps
+            color="primary"
```

Comment:
this one should probably be `outline` as well.
Just a reminder that the `outline` should also use the `primary` color instead of the other blue color.